### PR TITLE
[New Rule] - Defense Evasion IIS HttpLogging Disabled

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/141

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/defense_evasion_iis_httplogging_disabled.toml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/04/14"
+ecs_version = ["1.5.0"]
```
Comment (threat-punter) [inline rules/windows/defense_evasion_iis_httplogging_disabled.toml:3]:
```suggestion
creation_date = "2020/08/14"
ecs_version = ["1.6.0"]
```

File: rules/windows/defense_evasion_iis_httplogging_disabled.toml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+An attacker with IIS server access via a webshell or alike can disable HttpLogging as an effective anti-forensics
+measure. Verify the legitimacy context of this change.
```
Comment (threat-punter) [inline rules/windows/defense_evasion_iis_httplogging_disabled.toml:11]:
```suggestion
Identifies when Internet Information Services (IIS) HTTP Logging is disabled. An attacker with IIS server access via a webshell or other mechanism can disable HTTP Logging as an effective anti-forensics measure.
```

Comment (threat-punter) [reply rules/windows/defense_evasion_iis_httplogging_disabled.toml:11]: If you know what level of access is required to disable HTTP Logging on an IIS server, can we put that in the description of the rule too?

I'd recommend moving the sentence below to the notes field of the rule, as the description field is typically used to describe the behavior that the rule detects and the motive for why an attacker might execute the technique. The notes field contains information that's helpful for analysts to triage/investigate the alert.

> Verify the legitimacy context of this change.

File: rules/windows/defense_evasion_iis_httplogging_disabled.toml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+max_signals = 33
+name = "IIS HttpLogging Disabled"
```
Comment (threat-punter) [inline rules/windows/defense_evasion_iis_httplogging_disabled.toml:17]:
```suggestion
name = "IIS HTTP Logging Disabled"
```

**Review by threat-punter — CHANGES_REQUESTED** [review]
I really like this new rule. I left a few comments and suggestions for your review. 👍

File: rules/windows/defense_evasion_iis_httplogging_disabled.toml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Samirbous) [reply rules/windows/defense_evasion_iis_httplogging_disabled.toml]: agree, removed it (no value, description is quite clear logging disabled)

**Review by threat-punter — APPROVED** [review]

File: rules/windows/defense_evasion_iis_httplogging_disabled.toml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when Internet Information Services (IIS) HTTP Logging is disabled on a server. An attacker with IIS server
+access via a webshell or other mechanism can disable HTTP Logging as an effective anti-forensics measure.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
```
Comment (brokensound77) [inline rules/windows/defense_evasion_iis_httplogging_disabled.toml:13]:
```suggestion
from = "now-9m"
index = ["winlogbeat-*", "logs-endpoint.events.*"]
```

**Review by brokensound77 — APPROVED** [review]
Last comment, then LGTM 👍

**Pull request merged**